example/web-avatar-client/src/parseMMLDescription.ts

```diff
@@ -0,0 +1,86 @@
+import { MMLCharacterDescription, MMLCharacterDescriptionPart } from "./AvatarEditor";
+
+export type LoadingErrors = string[];
+
+export const parseMMLDescription = (
```

Comment:
This should be a library function in `3d-web-avatar`

Reply:
done